### DISCUSSION TITLE
Build failure on gradle assembleWebBeta task

### DISCUSSION THREAD

**Discussion by alanorth** [organicmaps/organicmaps#1346] [Q&A] [answered]
I'm trying to build an Android APK according to INSTALL.md. The `runWebDebug` task produces an APK, but I don't want a debug build. The docs say that this should work but it doesn't:

```console
$ ./gradlew clean assembleWebBeta
...
> Task :processWebBetaGoogleServices FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':processWebBetaGoogleServices'.
> No matching client found for package name 'app.organicmaps.beta'
```

In fact no other combination of build types and flavors work either, for example `assembleWebRelease`, `assembleRelease`. Is it something I'm doing wrong, or a bug in the gradle config? Thanks!

**Comment by biodranik** [comment, marked as answer]
Try to build assembleFdroidRelease (or Beta).

For other betas you need a google-services.json file with matching app id.

**Reply by alanorth** [reply]
Ah, that's it. Thanks. <3
